src/main/java/org/openmainframeproject/cobolcheck/features/testSuiteParser/Mock.java

```diff
@@ -165,4 +165,8 @@ private String getArgumentText(){
         }
         return combinedArgs.substring(0, combinedArgs.length() - 2);
     }
+
+    public void addLinesWithoutMoving(List<String> lines) {
```

Comment:
What lines is it that you don't want to move to area B?
Either way I think that overloading the `addLines` method with a `shouldMoveToAreaB` boolean would be more clean 😃